ci-operator/step-registry/ipi/conf/vsphere/check/ipi-conf-vsphere-check-commands.sh

```diff
@@ -116,8 +116,7 @@ source "${SHARED_DIR}/govc.sh"
 
 DATACENTERS=("$GOVC_DATACENTER")
 # If testing a zonal install, there are multiple datacenters that will need to be cleaned up
-# vlanid between 1287 and 1302 will use profile:vsphere-multizone-2
-if [ ${vlanid} -ge 1287 ] && [ ${vlanid} -le 1302 ]; then
+if [[ $JOB_NAME_SAFE == *"zones"* ]]; then
```

Comment:
@WenXinWei here is an example of how to check for a zonal job instead of using boskos and vlan ids - which will change.

Reply:
Personally I think this is another kinda of "hardcode", that limit ci job name definition, how about introduce a new var in cluster profile vault, e.g: `vsphere_datastore_extra: "datacenter-2"`, so that append it to `DATACENTERS` array for more flexibility.

Reply:
@jianlinliu I completely agree, I will work on that today